docs/dev/infrastructure.rst

```diff
@@ -1,86 +1,201 @@
-
-* astropy_
-* `specutils <https://specutils.readthedocs.io>`_
-* `spectral-cube <https://spectral-cube.readthedocs.io>`_ and/or `NDCube <https://docs.sunpy.org/projects/ndcube>`_
-* `photutils <https://photutils.readthedocs.io>`_
-
-Note that those libraries themselve depend on the wider scientific Python ecosystem, so this is by no means a comprehensive list of the libraries jdaviz depends on.  But these are the primary "top-level" libraries, meaning it is these that most users are likely to focus on to parallel or extend their jdaviz workflows.
-
-
-Selections
-==========
-
-This section explains the working theory behind the selection tool and was inspired by the the introduction of two methods to the SpecViz helper:
-`specviz.get_spectra()` and `specviz.get_spectral_regions()`
-
-Data loaded in are imported into jdaviz and immediately converted into a `specutils.SpectralRegion`. These are a spectral analog to the Astropy `regions` (which instead focuses on spatial regions and their associated WCS). These spectral regions are returned by the `specviz.get_spectra()` method.
-
-The selection tool allows the user to specify a specific range on the graph. This is defined by the underlying Glue Library upon which jdaviz relies on as a "Glue Subset." Thus throughout this section, we will refer to these user defined ranges as "subsets." Effectively, the selection tool defines a mask that can be thought of as "definition" of which data is and is not included in the subset. Upon extraction via `specviz.get_spectral_regions()`, the method will return a new `specutils.SpectralRegion` object that applies that mask atop of the proper region (data) displayed, and realizes the subset the user defined in jdaviz. 
-
-Related Documents
-=================
+* *Spectrum viewer*: A widget that shows a 1D astronomical data set, primarily aimed at
+  astronomical spectra. Interactivity includes panning, zooming, and region marking.
+* *Image viewer*: A widget that shows an astronomical image, along with its on-sky
+  coordinates when WCS are available. Interactivity includes panning, zooming, stretch
+  (contrast and scale), and cut values.
+* *Cube slicer*: A widget for displaying slices or similar aggregate operations on
+  spectroscopic data cubes. While similar to the image viewer in appearence and
+  interactive capabilities, the core difference is that the main data object is
+  expected to be a data cube rather than a 2D image, and this is reflected in additional
+  aggregation/slicing operations.
+* *Table viewer*: A widget to show tabular datasets like `astropy.table.Table` objects.
+  Primarily meant to be combined with other viewers to examine the complete set of
+  properties from a selection made in another viewer. Interactivty focuses on sorting
+  and selection of specific rows (to then be highlighted in other viewers or interacted
+  with in notebook/lab).
+
+In addition to the componenent widgets above, there are also *plugins* that goes with
```

Comment:
"that goes with" should be "that go with"

Reply:
Fixed.

Thanks for all the suggestions. Good catch!